paddle/pir/core/operation.cc

```diff
@@ -32,12 +32,18 @@ using detail::OpOutlineResultImpl;
 using detail::OpResultImpl;
 
 Operation *Operation::Create(const OperationArgument &argument) {
```

Comment:
```suggestion
Operation *Operation::Create(OperationArgument && argument) {
```
如果会修改argument的成员，这里还是直接传入右值比较合理吧？

Reply:
感谢，已经按要求修改为了右值。